pyiron_gpl/pacemaker/pacemaker.py

```diff
@@ -0,0 +1,290 @@
+# coding: utf-8
+# Copyright (c) ICAMS, Ruhr University Bochum, 2021
+# Distributed under the terms of "GPLv3", see the LICENSE file.
+
+import logging
+import numpy as np
+import os
+import pandas as pd
+import re
+import ruamel.yaml as yaml
+from pyiron_base import GenericJob, InputList
+from pyiron_base.generic.util import ImportAlarm
+from pyiron_base.settings.generic import Settings
```

Comment:
```suggestion
from pyiron_base import GenericJob, InputList, ImportAlarm, Settings
```

Please do not bind to internal modules of `pyiron_base`